cypress.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
can you remove the `.DS_Store` file?

Reply:
I can't find this file anywhere

Reply:
if you navigate to the committed files in this pr, you can see it committed at the top of the diff

Reply:
done.